frontend/public/components/namespace.jsx

```diff
@@ -82,11 +92,41 @@ import {
 import { removeQueryArgument } from './utils/router';
 import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
 
+import NamespaceDropDown from './namespace-dropdown';
```

Comment:
nit, Dropdown shouldn't be camelCase.

Reply:
The app is very inconstant about this as both Dropdown and DropDown are widely used in variable names.  Good to know that Dropdown is correct.  Changed on next commit